reconcile/test/test_slack_usergroups.py

```diff
@@ -86,6 +89,35 @@ def test_get_pagerduty_username_slack_username(self):
         result = integ.get_pagerduty_name(user)
         self.assertEqual(result, 'pd')
 
+    @patch.object(integ, 'SlackApi', autospec=True)
+    @patch.object(queries, 'get_app_interface_settings', autospec=True)
+    @patch.object(queries, 'get_permissions', autospec=True)
+    def test_get_slack_map_return_expected(self, get_permissions,
+                                           get_app_interface_settings,
+                                           slackApi):
+        get_permissions.return_value = self.get_permissions_fixture()
+        slack_api_mock = create_autospec(SlackApi)
+        expected_slack_map = {
+            'coreos': {
+                'slack': slack_api_mock,
+                'managed_usergroups': [
+                    'app-sre-team',
+                    'app-sre-ic'
+                ]
+            }
+        }
+        result = integ.get_slack_map()
+        slackApi.assert_called_once()
```

Comment:
nit: snake case? A little bit odd that our linter doesn't pick up on that, but maybe disabled.